### PR TITLE
Blog post: 10 years of Scala.js (for February 5).

### PR DESCRIPTION
**Pull request by sjrd**
image

### CODE REVIEW

**Comment by sjrd** [comment]
Also cc/review by @gzm0

**Comment by sjrd** [comment]
> This is a nice write-up! That being said, I wonder if it could be possible to give some hints about what to expect in the coming years as well.

This was my intention all along, but I couldn't fit it in. It would have had to come last to fit the story, but then the final thank you's would have been shadowed by that. I did not want that, so I left it out in the end.

File: _posts/2023-02-05-ten-years-of-scala-js.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+## Building on solid ground
+
+By the time we released version 0.1 in November 2013, the fundamental design goals of Scala.js were already firmly established:
+
+* interoperability with JavaScript: the ability to directly represent and manipulate JavaScript values as normal values from Scala.js,
+* portability with respect to Scala/JVM: compiling the semantics of Scala/JVM as accurately as possible to JavaScript, with [very few compromises](https://www.scala-js.org/doc/semantics.html) for the sake of interoperability, and
+* compatibility with IDEs, build tools, and other tooling: every valid Scala.js program must also typecheck as a valid Scala/JVM program, not to confuse tooling.
+
+These core design goals still govern the overarching decisions about the evolution of Scala.js today.
+An additional goal of good run-time performance emerged during the 0.5.x era, in 2014.
```
Comment (SethTisue) [inline _posts/2023-02-05-ten-years-of-scala-js.md:36]: (Maybe also mention reducing code size? Not sure.)

File: _posts/2023-02-05-ten-years-of-scala-js.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Scala.js was one of the first large projects in the Scala ecosystem to commit to multi-year backward binary compatibility, along with Akka and sbt.
+Version 0.6.0 had been built as a stepping stone for v1, with an expected lifetime of 6 months.
+Instead, it evolved while staying backward binary compatible for 5 years, before yielding to Scala.js 1.0.0 in February 2020.
+
+With such long compatibility windows, eventually Scala.js outlives the support lifetime of the underlying Scala version lines.
+Recently, we dropped support for Scala 2.11, which had been declared end-of-life 5 years earlier.
+Ironically, [the corresponding request for comments issue](https://github.com/scala-js/scala-js/issues/4759) was by far the most upvoted issue of the Scala.js issue tracker, with more than 100 upvotes.
+
+Scala.js is also known for being rock solid, owing to a strong code quality and testing discipline.
+Since the release of Scala.js 1.7.0 in August 2021, every version ships with zero known bug.
```
[the remark this answers is not in the document]
Comment (sjrd) [reply _posts/2023-02-05-ten-years-of-scala-js.md]: Hah! It still puzzles me that English considers 0 to be plural. :sweat_smile:

**Review by SethTisue — APPROVED** [review]

**Review by gzm0 — APPROVED** [review]
LGTM :)

File: _posts/2023-02-05-ten-years-of-scala-js.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+When we first started developing Scala.js, it was a one-person project at [LAMP, EPFL](https://www.epfl.ch/labs/lamp/)--the research lab led by Martin Odersky.
+
+Generating JavaScript from Scala was not a new idea.
+There had already been several attempts at doing so, both at LAMP ([Scala+GWT](https://scalagwt.github.io/), [JS Scala](https://infoscience.epfl.ch/record/179888/)) and in the community ([s2js](https://github.com/alvaroc1/s2js), [JScala](https://github.com/nau/jscala)).
+It felt like Mission Impossible at the time, which was exactly the motivation we needed to get started.
+
+Scala.js was met with enthusiasm when it was first shown off at Scala Days in June 2013--despite a 16 MB "Hello world" output.
+A small group of early enthusiasts gathered on a mailing list and started exchanging their pet projects, providing valuable early feedback to the project.
+The one-person project quickly grew into a two-person core team along with a few dozens of enthusiasts.
```
Comment (Philippus) [inline _posts/2023-02-05-ten-years-of-scala-js.md:25]:
```suggestion
The one-person project quickly grew into a two-person core team along with a few dozen of enthusiasts.
```

Comment (SethTisue) [reply _posts/2023-02-05-ten-years-of-scala-js.md:25]: it's even more idiomatic if you drop the "of": "a few dozen enthusiasts"

**Pull request merged**